### PR TITLE
add prepend-global task

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce PTAL. i'll migrate our internal release tasks to use this and anyone external can also use this

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ before_script:
   - npm install -g gulp
   - pip install --user protobuf
 script:
-  - npm run node-test
+  - npm run ava
```
Comment (cramforce) [inline .travis.yml]: Looks like a typo :)

**Comment by cramforce** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (erwinmombay) [inline .travis.yml]: my guess is you meant "java" 😹

**Pull request merged**